### ISSUE TITLE
PMC warning about the decimal Price { get; set; }

### ISSUE THREAD

**Issue by shadinamrouti** [aspnet/Docs#10402]
PMC shows the following warning when adding a migration. It seems a data annotation is required for the decimal Price.

> Microsoft.EntityFrameworkCore.Model.Validation[30000]
>       No type was specified for the decimal column 'Price' on entity type 'Movie'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'.

Suggested fix is to use the following annotation:

```
[Column(TypeName = "decimal(18,2)")]
public decimal Price { get; set; }
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1d975a83-1550-b284-d0d8-7fe2ea206ff5
* Version Independent ID: 48e81b42-7c2b-afbf-71e6-d4f442a4bd1f
* Content: [Add a model to an ASP.NET Core MVC app](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/adding-model?view=aspnetcore-2.2&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/first-mvc-app/adding-model.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/first-mvc-app/adding-model.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Webbarrr** [comment]
I'm confused as to why this was closed without an answer - I've had had this warning myself & was quite confused about whether or not I should progress without fixing it. A response would be appreciated

**Comment by shadinamrouti** [comment]
@Webarrr me too confused. I have a hope that one day Microsoft will issue a policy not to close tickets without a single reply because this kind of an action is passive-aggressive.

**Comment by Rick-Anderson** [comment]
@shadinamrouti 	@Webbarrr 
See Rick-Anderson closed this in #10412 22 hours ago
That means it's fixed. Click on #10412 to see the resolution.
The PR closed the issue automatically. That's how Github works.
https://github.com/aspnet/Docs/pull/10412/files
